src/js/enactors.js

```diff
@@ -0,0 +1,325 @@
+        content.eq(0).prepend(navToggle);
+    }
+};
+
+uioPlus.enactor.simplify.setVisible = function (that, node, mutationRecord) {
+    var elm = node.nodeType === Node.ELEMENT_NODE ? $(node) : $(mutationRecord.target);
+
+    var makeVisible = fluid.find(that.options.alwaysVisible, function (selectorName) {
+        return elm.is(that.options.selectors[selectorName]) || undefined;
+    }, false);
+
+    if (makeVisible) {
+        elm.css("visibility", "visible");
+    }
+};
+
+uioPlus.enactor.simplify.set = function (that, state) {
+    if (state && that.content.length) {
+        that.content.css("visibility", "visible");
+        fluid.each(that.options.alwaysVisible, function (selector) {
+            that.locate(selector).css("visibility", "visible");
+        });
+        that.container.css("visibility", "hidden");
+        if (that.options.injectNavToggle) {
+            uioPlus.enactor.simplify.injectToggle(that, that.content);
+        }
+        that.locate("navToggle").show();
+        that.observer.observe();
+    } else if (that.content.length) {
+        that.locate("navToggle").hide();
+        that.container.css("visibility", "");
+        that.content.css("visibility", "");
+        fluid.each(that.options.alwaysVisible, function (selector) {
+            that.locate(selector).css("visibility", "");
+        });
+        that.observer.disconnect();
+    }
+};
+
+uioPlus.enactor.simplify.toggleNav = function (that, state) {
+    var navToggle = that.locate("navToggle");
+    if (state && that.model.simplify) {
+        that.nav.css("visibility", "visible");
+        navToggle.attr("aria-pressed", state);
+    } else {
+        that.nav.css("visibility", "");
+        navToggle.attr("aria-pressed", state);
+    }
+};
+
+/******************************************
+ * table of contents enactor              *
+ ******************************************/
+
+fluid.defaults("uioPlus.enactor.tableOfContents", {
+    gradeNames: ["uioPlus.contentView", "fluid.prefs.enactor.tableOfContents"],
+    selectors: {
+        tocContainer: ".flc-toc-tocContainer",
+        article: "article, [role~='article'], .article, #article",
+        main: "main, [role~='main'], .main, #main",
+        genericContent: ".content, #content, .body:not('body'), #body:not('body')"
+    },
+    defaultContent: "{that}.container",
+    listeners: {
+        "onCreateTOCReady.injectToCContainer": {
+            listener: "{that}.injectToCContainer",
+            priority: "first"
+        }
+    },
+    invokers: {
+        injectToCContainer: {
+            funcName: "uioPlus.enactor.tableOfContents.injectToCContainer",
+            args: ["{that}"]
+        }
+    },
+    markup: {
+        tocContainer: "<div class=\"flc-toc-tocContainer uioPlus-toc-tocContainer\"></div>"
+    }
+});
+
+uioPlus.enactor.tableOfContents.injectToCContainer = function (that) {
+    if (!that.locate("tocContainer").length) {
+        if (that.content.length === 1) {
```

Comment:
A comment explaining why the variant strategy is necessary here would be helpful